backend/database/versions/b65dad1f2a7e_add_explorer_field_to_dataset.py

```diff
@@ -0,0 +1,52 @@
+"""add_explorer_field_to_dataset
+
+Revision ID: b65dad1f2a7e
+Revises: 600689e11cf4
+Create Date: 2021-06-02 14:52:17.931804
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "b65dad1f2a7e"
+down_revision = "78103925d379"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("dataset", sa.Column("explorer_s3_uri", sa.String(), nullable=True))
+    op.add_column("dataset", sa.Column("explorer_url", sa.String(), nullable=True))
+    op.create_index(op.f("ix_dataset_explorer_url"), "dataset", ["explorer_url"], unique=False)
+    op.execute("UPDATE dataset SET explorer_url=deployment_directory.url FROM deployment_directory WHERE deployment_directory.dataset_id=dataset.id;")  # noqa E501
+    op.execute("UPDATE dataset SET explorer_s3_uri=CONCAT('s3://hosted-cellxgene-', split_part(explorer_url, '.', 2), '/', split_part(explorer_url, '/', 5), '/') WHERE dataset.explorer_url is not null")  # noqa E501
```

Comment:
This isn't going to work for prod no? There's no `prod` value in the cellxgene.cziscience.com.